packages/flutter/lib/src/services/text_input.dart

```diff
@@ -1016,6 +1016,16 @@ enum SelectionChangedCause {
   scribble,
 }
 
+/// An enumeration of the status of the dragged handle.
```

Comment:
how about: 
"An enum to differentiate between handles.

Used when an action may have different results based on the handle side."